action.yml

```
@@ -19,6 +19,10 @@ inputs:
       Reporter of reviewdog command [github-pr-check,github-pr-review].
       Default is github-pr-check.
     default: 'github-pr-check'
+  rubocop_version:
+    description: 'Rubocop version'
+  rubocop_extensions:
+    description: 'Rubocop extensions'
```

```suggestion
    description: 'Rubocop extensions'
    default: 'rubocop-rails rubocop-performance rubocop-rspec rubocop-i18n rubocop-rake'
```
Please use the `default` option to keep the list of extensions which install by default.